_source/_posts/2023-06-29-devday23-beyond-mvp.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: blog_post
+title: "How to Take Your Enterprise-Ready SaaS App Beyond MVP"
+author: alisa-duncan
+by: advocate
+communities: [devops,security,.net,java,javascript,php,python]
+description: "With enterprise-ready identity added to your SaaS app, you're ready to take your app beyond MVP! Find out how you can make your app stand out, and get a closer look at provisioning and Okta workflows with Aaron Parecki, Security Architect Group Manager at Okta."
+tags: [devday23, enterprise-ready]
+tweets:
+- ""
+image: blog/devday23-beyond-mvp/social.jpg
+type: awareness
+---
+
+Ready to move beyond MVP in the journey of adding enterprise-ready identity in your SaaS app? With the must-have functionality in place, you're ready to make your app stand out, get noticed by enterprise customers, and handle user provisioning and automation that can scale!
+
+So what might that look like? 🤔
+
+In this installment of creating an enterprise-ready SaaS app, we'll cover this and more!
+
+|Posts in the enterprise-ready Devday23 presentation series|
+| --- |
+| 1. [What Does It Mean to Be Enterprise Ready: Devday23 Keynote Enterprise Readiness](/blog/2023/06/13/devday23-enterprise-ready-keynote) |
+| 2. [What Enterprises Look For: Devday23 Enterprise Customer Perspectives](/blog/2023/06/15/devday23-enterprise-customer) |
+| 3. [How to Build an Enterprise-Ready MVP SaaS App]() |
+| 4. **How to Take Your Enterprise-Ready SaaS App Beyond MVP** |
+
+## Take your enterprise-ready SaaS app to the next level
+
+Aaron Parecki, Security Architect Group Manager at Okta, shared ways you can level up your SaaS app at [WIC Devday 23](https://developerday.com/events/devday23-wic) virtual conference:
+
+{% youtube XCHVUPEKxs4 %} 
+
+## Learn more about Okta Workflows, provisioning, and SaaS application maturity
+
+{% speakerdeck ee97e681e84646879cfb303bfa061604 600px %}
+
```

Comment:
* [SCIM](https://scim.cloud)
* [A Developer's Guide to SCIM](https://youtu.be/JmA83cy0uVc) (video)
* [Okta Workflow Connector Builder](https://help.okta.com/wf/en-us/Content/Topics/Workflows/connector-builder/connector-builder.htm)
* [Workflow Tutorials](https://www.youtube.com/playlist?list=PLIid085fSVdvyK8F4xuk49EchBPmAVNHG) (video)